agent/api/ecsclient/client.go

```diff
@@ -288,13 +288,13 @@ func (client *APIECSClient) getCustomAttributes() []*ecs.Attribute {
 
 func (client *APIECSClient) SubmitTaskStateChange(change api.TaskStateChange) error {
 	// Submit attachment state change
-	if change.Attachments != nil {
+	if change.Attachment != nil {
```

Comment:
something that wasnt clear to me and probably included in a different PR - where are these eni attachment events being emitted from? is it only out of the `eni/watcher` package?